src/Apps/Order2/Routes/Details/Components/Order2DetailsFulfillmentInfo.tsx

```diff
@@ -49,32 +62,32 @@ const getShippingContent = (fulfillmentDetails): React.ReactNode => {
   return (
     <>
       {name && (
-        <Text variant="xs" color="mono100">
+        <Text variant={["xs", "sm"]} color="mono100">
```

Comment:
Minor: Do we want `[xs, sm]` or `[xs, xs, sm]`? On checkout i feel like it was that 3rd breakpoint where we went into desktop mode. But I might be wrong here. (this would apply to all responsive arrays).